src/jit/gentree.h

```diff
@@ -1916,6 +1916,9 @@ class GenTreeUseEdgeIterator final
 #ifdef FEATURE_SIMD
     void MoveToNextSIMDUseEdge();
```

Comment:
I am hopeful that we can eliminate this, but I'm not convinced. These functions are also used by `GenTree::Operands()`, and I can imagine that there are situations in which we want to iterate over the operands to a call but not expand multi-reg args. It might be worth eliminating the parameter until those cases have been found, but that should be addressed in a separate change.